db/schema.rb

```diff
@@ -997,8 +1002,13 @@
     t.datetime "reset_password_sent_at"
     t.string   "api_key",                :limit => 40
     t.integer  "enterprise_limit",                     :default => 1, :null => false
+    t.string   "confirmation_token"
+    t.datetime "confirmed_at"
+    t.datetime "confirmation_sent_at"
+    t.string   "unconfirmed_email"
   end
 
+  add_index "spree_users", ["confirmation_token"], :name => "index_spree_users_on_confirmation_token", :unique => true
```

Comment:
It seems reasonable that we query only this column sometimes (I guess from "cofirm your email" link type) but I'd like to see where. Indexes do not come for free, they do bring a cost on writes to the db (they must be updated), so better to have one that fits our needs perfectly.